### ISSUE TITLE
Coldcard Mk3 gives errors when attempting to sign multisig send bitcoin

### ISSUE THREAD

**Issue by silicus26** [cryptoadvance/specter-desktop#1501]
I sent the following issues to Coinkite and they said to check with the Specter Telegram group. I posted there but figured I'd post here, too. 

I am attempting to sign a Mulitsig 2-of-3 send bitcoin transaction. I signed just fine with one wallet. But I’m having issues with my ColdCard Mk3 wallet.

I updated my Specter Desktop, that created the Multisig Wallet, to 1.7.2.

I updated the ColdCard Mk3 firmware from 3.2.2 to 4.1.3 also during this.

I am on Bitcoin Core 0.21.1..

Here are the issues I get when trying to sign the transaction.

With MicroSD:
Failure
My XFP not 
Involved

With Direct USB(HWI):
Selected device doesn't have matching signing key.

Any ideas on how to fix this issue?

I see someone fixed a similar issue by deleting the multisig and recreating the multisig wallet with the very same multi-sig file. (source: https://github.com/cryptoadvance/specter-desktop/issues/1373 TITILE: Cannot sign multisig using coldcard #1373) 

I have to figure out how to do that without losing my bitcoin!

**Comment by silicus26** [comment]
I was told to try deleting/reloading multisig to Coldcard Mk3.  I deleted my multisig off the Coldcard then attempted to import multisig from Specter and get this error: "Failed to import. My key not included multisig.py:691". 

I tried importing XPUB from ColdCard to my multisig on Specter and then exporting multisig file again to Coldcard and get same error. I will post this to the GitHub issue page.

When initially setting all this up I used the following guide where I also have a Cobo Vault (that works fine with multisig) and a Paper wallet. https://btcguide.github.io/setup-wallets/

**Comment by silicus26** [comment]
I will attempt a workaround, tomorrow, with some help. If that works, I'll mention what I did here and close this issue.

**Comment by k9ert** [comment]
I'm happy to try to reproduce but would be good to have more information here: At least the derivation pathes used in the multisig and which devices they have been created on.
I'm not sure how i can help otherwise as the problem is on the signing device.

**Comment by silicus26** [comment]
I think I know what happened. I forgot that I included a BIP-39 Passphrase. At least I wrote it down and the signature that should appear.

**Comment by silicus26** [comment]
I now realize my goof. I created a BIP-39 passphrase on the Coldcard and forgot to put that in the steps of using that wallet. At least I wrote down that passphrase and the signature it should show after entry. That should solve my problems. Otherwise, I have a workaround now.

 

Thanks for all the help! I feel terrible for leaving that info out of my step-by-step guide. For myself (and close family).
